logmiddleware.go

```diff
@@ -61,15 +61,47 @@ func RequestMiddlewareLogger(logger *logrus.Logger, excludedPrefix []string) mux
 			}
 
 			Get(ctx).WithFields(logrus.Fields{
-				"method":   r.Method,
-				"url":      r.URL.String(),
-				"hostname": r.URL.Hostname(),
+				"http":	map[string]interface{}{
+					"request": map[string]interface{}{
+						"method": r.Method,
+						"userAgent": map[string]interface{}{
+							"original": r.Header.Get("user-agent"),
+						},
+					},
+				},
+				"url": map[string]interface{}{
+					"path": r.URL.String(),
+				},
+				"host": map[string]interface{}{
+					"hostname": r.URL.Hostname(),
+					"ip":  r.Header.Get("x-forwaded-for"),
```

Comment:
@davidebianchi do we forward this header to the services? I think we do, so this should be ok.

Reply:
Yes, this should be forwarded

Reply:
ok